packages/browser-extension/src/scripts/background.js

```diff
@@ -6,7 +6,9 @@ if (typeof browser === 'undefined') {
  * @description API URL
  * @type {string}
  */
-const apiUrl = 'https://api.cookie-dialog-monster.com/rest/v3';
+
+//const apiUrl = 'https://api.cookie-dialog-monster.com/rest/v3';
+const apiUrl = 'http://localhost:8080/rest/v3';
```

Comment:
Please, these changes to test things in your development environment should be reverted before uploading a PR.

Reply:
ok

Reply:
done